src/pages/event.js

```diff
@@ -190,77 +218,71 @@ export function Event() {
-              <h4 style={{marginBottom: '0'}}><div className='event-title'>EVENT ID</div><div className='event-id'>#{eventId}</div> </h4>
-              <a href={parseInt(eventId)+1} >{'Next  '}<FontAwesomeIcon icon={faAngleRight}/></a>
-            </div>
-            <div style={{minHeight: '200px', margin: '0 auto'}}>
-              <EventCard key={0} event={event} size='l' power={power} />
+        {
+          loadingEvent === 'loading' || loadingEvent === 'idle' ?
+              <div style={{display: 'flex', justifyContent: 'center'}}>
+                <Loader />
+              </div> :
+          errorEvent || Object.keys(event).length === 0 ?
+            <div style={{display: 'flex', justifyContent: 'center', flexDirection: 'column', margin: '0 auto', textAlign: 'center'}}>
+              <h2>{errorEvent || 'Token not found'}</h2>
+              <div >
+                <img alt="warning sign" style={{maxWidth: '30rem'}} src="/icons/warning.svg"/>
+              </div>
+            </div> :
+            <div className="container">
+              <div style={{
+                display: 'flex',
+                alignItems: 'center',
+                flexWrap: 'wrap',
+                alignContent: 'space-around',
+                justifyContent: 'space-around',
+                marginBottom: 82,
+              }}>
+                <div style={{flex: '0 0 18rem', display: 'flex', flexDirection: "column", justifyContent: "center"}}>
+                  <div className='prev-next-buttons' style={{display: 'flex', justifyContent: 'space-between', marginBottom: 38,}}>
+                    <a href={parseInt(eventId)-1} ><FontAwesomeIcon icon={faAngleLeft}/>{'  Prev'}</a>
+                    <h4 style={{marginBottom: '0'}}><div className='event-title'>EVENT ID</div><div className='event-id'>#{eventId}</div> </h4>
+                    <a href={parseInt(eventId)+1} >{'Next  '}<FontAwesomeIcon icon={faAngleRight}/></a>
+                  </div>
+                  <div style={{minHeight: '200px', margin: '0 auto'}}>
+                    <EventCard key={0} event={event} size='l' power={power} />
+                  </div>
+                </div>
+              </div>
+              <div className='table-header'>
+                <div className='table-title'>Collections <span>({tokens.length})</span></div>
+                {csvReadyOrAlmostReady() ?
+                  <CSVLink
+                      filename={`${event.name}.csv`}
+                      target="_blank"
+                      data-tip={`${csvOnlyMissingEns() ? 'Please wait if you want the ens names too' : ''}`}
+                      className={'btn'}
+                      style={csvButtonStyle}
+                      data={csv_data}
+                  >
+                    <CSVLinkText>{`Download CSV${csvOnlyMissingEns() ? ' (without ens)' : ''}`}</CSVLinkText>
+                    <ReactTooltip effect={'solid'} />
+                  </CSVLink> :
```

Comment:
this double conditional with `:`, in this case, increases the cognitive load of the code and adds a high probability of error by not detecting that it is a double conditional. Be very careful in the future use of these conditionals, they are only advisable in cases of one line of code.
The most advisable thing now is to separate into two conditionals, even if it is the same but denied.